runners/flink/runner/src/test/java/org/apache/beam/runners/flink/streaming/TestCountingSource.java

```diff
@@ -0,0 +1,256 @@
+/*
```

Comment:
I wonder if you could use CountingSource.forever()/CountingInput.unbounded() instead?